server/src/test/java/io/crate/auth/UserAuthenticationMethodTest.java

```diff
@@ -361,6 +375,136 @@ public void test_jwt_authentication_claim_mismatch_throws_error() throws Excepti
+            // Matches JWT_TOKEN payload.
+            .put(AuthSettings.AUTH_HOST_BASED_JWT_ISS_SETTING.getKey(), "https://console.cratedb-dev.cloud/api/v2/meta/jwk/")
+            // Matches JWT_TOKEN payload.
+            .put(AuthSettings.AUTH_HOST_BASED_JWT_AUD_SETTING.getKey(), "test_cluster_id")
+            .build();
+        JWTAuthenticationMethod jwtAuth = new JWTAuthenticationMethod(
+            roles,
+            settings,
+            jwkProviderFunction(null),
+            () -> null // ClusterId supplier is not used, aud is taken from defaults
+        );
+
+        Credentials credentials = new Credentials(JWT_TOKEN);
+        credentials.setUsername(tokenUsername);
+
+        Role authenticatedRole = jwtAuth.authenticate(credentials, null);
+        assertThat(authenticatedRole.name()).isEqualTo(tokenUsername);
+    }
+
+    @Test
+    public void test_jwt_authentication_user_has_no_jwt_properties_aud_is_not_defined_in_default() throws Exception {
+        String tokenUsername = "cloud_user"; // Token payload dictates CrateDB username.
+        Roles roles = () -> List.of(
+            userOf(
+                tokenUsername,
+                Set.of(),
+                new HashSet<>(),
+                getSecureHash("pwd"),
+                // User doesn't have JWT properties
+                null
+            )
+        );
+        var settings = Settings.builder()
+            // issuer Matches JWT_TOKEN payload.
+            // No default value for aud, must use clusterId.
+            .put(AuthSettings.AUTH_HOST_BASED_JWT_ISS_SETTING.getKey(), "https://console.cratedb-dev.cloud/api/v2/meta/jwk/")
+            .build();
+        JWTAuthenticationMethod jwtAuth = new JWTAuthenticationMethod(
+            roles,
+            settings,
+            jwkProviderFunction(null),
+            () -> "test_cluster_id" // Default aud is not set, fallback to value from the token payload.
+        );
+
+        Credentials credentials = new Credentials(JWT_TOKEN);
+        credentials.setUsername(tokenUsername);
+
+        Role authenticatedRole = jwtAuth.authenticate(credentials, null);
+        assertThat(authenticatedRole.name()).isEqualTo(tokenUsername);
+    }
+
+    @Test
+    public void test_jwt_authentication_user_has_no_jwt_properties_default_is_not_set() throws Exception {
+        String tokenUsername = "cloud_user"; // Token payload dictates CrateDB username.
+        Roles roles = () -> List.of(
+            userOf(
+                tokenUsername,
+                Set.of(),
+                new HashSet<>(),
+                getSecureHash("pwd"),
+                // User doesn't have JWT properties
+                null
+            )
+        );
+        JWTAuthenticationMethod jwtAuth = new JWTAuthenticationMethod(
+            roles,
+            Settings.EMPTY,
+            jwkProviderFunction(null),
+            () -> "test_cluster_id" // Default aud is not set, fallback to value from the token payload.
```

Comment:
Is this comment really valuable and related to the test case?
```suggestion
            () -> "test_cluster_id" 
```